src/passes/typeNameRemover.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ElementaryTypeNameExpression,
+  ExpressionStatement,
+  getNodeType,
+  IndexAccess,
+  TypeNameType,
+  VariableDeclarationStatement,
+} from 'solc-typed-ast';
+import { AST } from '../ast/ast';
+import { ASTMapper } from '../ast/mapper';
+import assert from 'assert';
+import { Expression, TupleExpression } from 'solc-typed-ast';
+import { notNull } from '../utils/typeConstructs';
+import { expressionHasSideEffects } from '../utils/utils';
+
+export class TypeNameRemover extends ASTMapper {
+  visitExpressionStatement(node: ExpressionStatement, ast: AST): void {
+    if (
+      (node.vExpression instanceof IndexAccess ||
+        node.vExpression instanceof ElementaryTypeNameExpression) &&
```

Comment:
There can be more expression besides IndexAccess and ElementaryTypeNameExpression, which you'd want to cover. 
e.g
```solidity
library D {
    enum A{a}
    struct B{
        A a;
    }
}
contract C {
    function f(bytes calldata _x) public pure{
        D.A;
        C;
        D;
        D.B;
    }
}
```

Reply:
Could you handle one more case, if these type of expressions are being used inside TupleExpression for e.g `(C, D.A, 2)` should also be disregarded.